### ISSUE TITLE
Fabric Website: Add support for versioned documentation

### ISSUE THREAD

**Issue by friendbool** [OfficeDev/office-ui-fabric-react#7269]
I'm doing development within SPFx.
Is there somewhere documentation for v.5?

_Originally posted by @cliffkoh in https://github.com/OfficeDev/office-ui-fabric-react/issues/6858#issuecomment-433995571_

**Comment by dzearing** [comment]
@Jahnp Is there any way we could deploy the doc site for v5?

**Comment by friendbool** [comment]
Thanks guys
I await news

**Comment by Jahnp** [comment]
For sure! Off the top of my head, it will probably require the following:

1. Modifying the website on the v5 branch to use the v6 pipeline format
1. Re-tooling the v5 build to publish the website from that branch
1. Setting up an endpoint for the v5 website to consume it
1. Setting up a v5 release to automate deployment

There's probably more involved and some details I'm missing. Will try to carve out time to investigate & confirm work required over the next few days.

**Comment by cliffkoh** [comment]
Oh, step 1 right there is going to be a lot of work... It would unblock deployment of the dev site to aka.ms/fabricdemo though.

**Comment by Jahnp** [comment]
@cliffkoh It might not actually be that bad. I *think* we would basically just need to port the changes from #6683 to the 5.0 branch. Haven't dug much deeper than a cursory diff but I think that should do it for all future 5.x releases.

**Comment by dzearing** [comment]
Sure thing. Closing the issue, given you can access the branch.

**Comment by Jahnp** [comment]
After some discussion, we're plan to add support for versioned documentation on the Fabric website. In the beginning, this will allow you to switch between docs for the latest v5 and v6 releases. We'll shoot to have this ready in February.

**Comment by Jahnp** [comment]
Oops, accidentally closed--sorry. Was going to say that @leddie24 and I went over the technical plan for this and will begin prototyping work shortly.

**Comment by Jahnp** [comment]
We're close to finishing this work, but have some outstanding work items to finish. We'll go ahead and move this to March--we expect to be finish within the next 1-2 weeks.

**Comment by Jahnp** [comment]
Update: This has been launched!

You can now access Fabric v5 docs from the [Fabric homepage](http://uifabric.io):
![image](https://user-images.githubusercontent.com/307118/55436526-65ca9d00-5551-11e9-9c38-3ae81e1a60df.png)

Please give this a spin and let us know if you have any issues or questions. We do know that there are some visual bugs in the left nav--we'll track those in a separate issue as we don't believe they're scenario-blocking.

Also, note that the approach we used here is a short-term solution that enables the basic version-switching functionality, but isn't super extensible (e.g. switching between minor & patch versions). In the future, we'll be revisiting the deployment/releaes pipeline to better load & manage releases more granularly.

**Comment by Jahnp** [comment]
Also, massive kudos to @leddie24 for enabling the bulk of this feature!

**Comment by Jahnp** [comment]
Closing this for now as a working version of this is complete. We'll continue to invest in more robust version-switching over time.
